aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -390,6 +390,22 @@ public void observe(
         onObservationFailure.accept(new DataStoreException("Not implemented yet, buster!", "Check back later!"));
     }
 
+    /**
+     * Stops all synchronization processes and deletes the
+     * SQLite database from device's local storage.
+     * @param onComplete Invoked if the call is successful.
+     * @param onError Invoked if an exception occurs.
```

Comment:
Seems like this should be changed to: "Invoked if not successful" ?

Reply:
Yeah...that makes more sense. Updating it.